### DISCUSSION TITLE
Gregorian music

### DISCUSSION THREAD

**Discussion by 68pandreas** [Audiveris/audiveris#616] [Q&A] [answered]
Hi, I tried to use a piece of printed gregorian music using the typical four instead of five lines and the C and F key on or between any of them; obviously this isn't supported, is it? Is there any tool known which could be used on this kind of music notation or is it planned to support this? Best regards, Pandreas

**Comment by hbitteur** [comment]
Right now, 4-line staves should be detected but considered as tablatures (for bass guitar) and their area ignored for further processing.
So, I'm afraid this is not the kind of "support" you would expect.

Could you post or point to an example of such gregorian music, so that I can see precisely what you mean?

**Reply by 68pandreas** [reply]
Hi, 
something like seen here:
https://3.bp.blogspot.com/-XRcMXlh7tes/TZO4yi0_2cI/AAAAAAAAADM/YGxwZBF7Md0/s1600/palmso_Introitus.jpg
If I knew an email I could also send some scans of a book by wetransfer.

**Comment by hbitteur** [comment, marked as answer]
I see!
This is really too far from the "Common western music notation" we address in Audiveris.
And there is no plan for this. 
Sorry.

**Reply by 68pandreas** [reply]
OK, Thanks!
